x-pack/platform/plugins/shared/alerting/server/routes/rule/apis/bulk_enable/bulk_enable_rules_route.test.ts

```diff
@@ -281,4 +282,41 @@ describe('bulkEnableRulesRoute', () => {
       });
     });
   });
+
+  describe('internally managed rule types', () => {
+    it('throws 400 if the rule type is internally managed', async () => {
```

Comment:
Reading these for the different APIs, I noticed you don't test the `"Rule type not found"` error path. I think it is fine, but I just wanna confirm. For each of these routes, the `rulesClient` does that check too, right?